index.html

```diff
@@ -5,12 +5,78 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <title>Responsive Cake Webpage</title>
+    <link
+    href="//fonts.googleapis.com/css?family=Roboto:400,500,300"
+    rel="stylesheet"
+    type="text/css"
+    />
+    <link rel="stylesheet" href="/style/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="/img/red.jpg"/>
+    <!-- <script src="https://kit.fontawesome.com/c0b874b18b.js" crossorigin="anonymous"></script> -->
+
 </head>
 
 <body>
-    <!-- Add your markup here -->
-</body>
+    <header class="header">
+        <!-- <img src="/img/Cupcake.png" alt="" class="header-pic"> -->
+        <nav class="nav-header">
+            <li href="#">HOME</li>
+            <li href="#">CAKES</li>
+            <li href="#">Ordering</li>
+            <li href="#">LESSON</li>
+            <li href="#">ABOUT</li>
+        </nav>
+        <div class="add">
+            <p class="add-text">
+                The best cakes in town delivered to you door.
+            </p>
+        </div>
+    </header>
+
+    <div class="nav2">
+        <nav class="nav-body">
+            <ul class="nav-li">
+                <li href="#">HOME</li>
+                <li href="#">CAKES</li>
+                <li href="#">Ordering</li>
+                <li href="#">LESSON</li>
+                <li href="#">ABOUT</li>
+            </ul>    
+        </nav>
+    </div>
 
-</html>
\ No newline at end of file
+    <section class="homepage-main"> 
+        <div class="main-hero">
+            <h1 class="title">Welcome</h1>
+            <p class="text-hero">It’s really not a special occasion without a generous slice of cake or two.
+                Next birthday, don’t settle for a Colin the Caterpillar (as good as they are); 
+                instead treat yourself or a lucky friend to an indulgent, 
+                bespoke show-stopper. Beautifully iced, exquisitely flavoured 
+                and made just to your requirements, these are birthday cakes 
+                to remember from some of London's most talented bakers.
+            </p>
+        </div>
+        <div class="wrapper-img">
+            <div class="img-hero" >
+                <img src="/img/strawberrry cake.jpg" alt="wedding strawberry cake">
+            </div>
+        </div>
+        <div class="products">  
+            <div class="homepage-products">
+                <img src="/img/wedding.avif" alt="wedding cake" class="wedding">
```

Comment:
This isn't a big deal for this exercise, but just including the info out of interest - 
.avif image format isn't supported by all main browsers; .webp might be a better choice as it also has very good small file sizes and wide browser support.